ci/testkomodo.sh

```diff
@@ -117,16 +112,12 @@ start_tests() {
 
     set +e
 
+    export ERT_PYTEST_ARGS=--eclipse-simulator
+
     # Run all ert tests except tests evaluating memory consumption and tests requiring windows manager (GUI tests)
-    pytest --eclipse-simulator -n auto --show-capture=stderr -v --max-worker-restart 0 \
-        -m "not limit_memory and not requires_window_manager" --benchmark-disable --dist loadgroup
+    just check-all
```

Comment:
Do we have this installed on the nodes?

Reply:
just will be installed with `pip install -e ".[dev]"`

Reply:
Installed when installing ert.